### DISCUSSION TITLE
Kernel crashing when evaluating functions made with ufuncify.

### DISCUSSION THREAD

**Discussion by MaticMl** [sympy/sympy#26591] [Q&A]
Hello!
I am using sympy to compute some very long and nonlinear dynamics equations using the LagrangesMethod class. I then use ufuncify to create numerical functions as lambdified functions take upwards of 1s to evalute. Using ufuncify works for some expressions which are a bit shorter, but for some very long expressions the kernel crashes while evaluating. Ufuncify does throw any errors while creating the wrappers, even with verbose set to True. Is there any flags I can pass to the backend to fix this issue.
Thank you for the help!

**Comment by moorepants** [comment]
I would recommend inspecting the C code that is generated by ufuncify.

**Reply by moorepants** [reply]
Also if you can provide a minimal example here we can potentially help debug.

**Reply by MaticMl** [reply]
Hi, I was unable to create a minimal example that would exhibit the same behaviour as my issue. The generated C code looks normal and I am unable to see the difference between a wrapper module that works, and one that crashes the kernel. It may be a memory related issue.
For anyone else experiencing this issue, I was able to sufficiently speed up the numerical functions by using the ```cse=True``` flag in ```sympy.lambdify()```. This brings the execution time down to the same order of magnitude as ufuncify and autowrap in my case.
Thank you for your help!
